<launch>

  <node pkg="convex_optimize_fitting" type="convex_optimize_fitting" name="convex_optimize_fitting" output="screen" >
    <param name="input_cluster_topic"           value="/segmentation/autoware_objects_array"/>
    <param name="bbox_target_frame"           value="velodyne"/>     
  </node>

  <!-- Rviz -->
  <node type="rviz" name="rviz" pkg="rviz" args="-d $(find convex_optimize_fitting)/rviz/convex_optimize_fitting.rviz" output="log" respawn="false" />

</launch>